index.html

```diff
@@ -14,7 +14,10 @@
     <script src="bower_components/momentjs/min/moment.min.js"></script>
     <script src="bower_components/momentjs/min/locales.min.js"></script>
     <script src="bower_components/humanize-duration/humanize-duration.js"></script>
-    <script src="dist/angular-timer.js"></script>
+    <!--<script src="dist/angular-timer.js"></script>-->
```

Comment:
@siddii I think I am going to create a angular-timer-all.js for index.html that could be used in the gh-pages and for https://github.com/siddii/angular-timer/issues/168